Templates/DLS.ini

```diff
@@ -11,15 +11,15 @@ DisabledControls=80,86,27
 
 # Extra patch
 ; Enables or disables the extra patch (Default = false)
-ExtraPatch=true
+ExtraPatch=false
```

Comment:
Considering `AdvancedHookV.dll` will be packaged with DLSv2 on release, maybe we can consider keeping the ExtraPatch enabled by default as all users will have the ability to use it.

Reply:
what does the extra patch do?

Reply:
Allows for toggling extras without having to fix the vehicle.

Reply:
i would maybe even make it a required thing, as i cant think of a reason not to use it